meteor/server/api/rest/v1/playlists.ts

```diff
@@ -786,6 +786,27 @@ export function registerRoutes(registerRoute: APIRegisterHook<PlaylistsRestAPI>)
 		}
 	)
 
+	registerRoute<{ playlistId: string; sourceLayerIds: string }, never, void>(
+		'delete',
+		'/playlists/:playlistId/sourceLayers/:sourceLayerIds',
+		new Map([
+			[404, [UserErrorMessage.RundownPlaylistNotFound]],
+			[412, [UserErrorMessage.InactiveRundown]],
+		]),
+		playlistsAPIFactory,
+		async (serverAPI, connection, event, params, _) => {
+			const playlistId = protectString<RundownPlaylistId>(params.playlistId)
+			const sourceLayerIdsStr = params.sourceLayerIds
+			logger.info(`API DELETE: sourceLayers ${playlistId} ${sourceLayerIdsStr}`)
+
+			check(playlistId, String)
+			check(sourceLayerIdsStr, String)
+
+			const sourceLayerIds = sourceLayerIdsStr.split(',')
```

Comment:
The comma-separated array format for URL params, is that a first for this API? Do we need explicit description of the expected format of the input in the API?

Also, in terms of REST, are we sure this goes as a param and not in the request body?

Reply:
I think it is a first, I would expect that the syntax shouldn't be too unsurprising as comma separated is an openapi convention. https://swagger.io/docs/specification/serialization/

I don't know if it would make more sense in the body, I was trying to keep it match the existing api structure which doesn't make much use of the body

Reply:
Yeah, I'm not questioning the format, just if we need to document how the param is intended to be used.

And for the question about the body, I don't have any opinion.

Reply:
following other changes, this is now a body parameter